wfl/App_Start/WebApiConfig.cs

```diff
@@ -1,17 +1,21 @@
 ﻿using System.Web.Http;
+using Drum;
+using Hypermedia;
+using Hypermedia.Transforms;
+using Hypermedia.Siren;
+using Symphono.Wfl.Models;
 
 namespace Symphono.Wfl
 {
     public static class WebApiConfig
     {
         public static void Register(HttpConfiguration config)
         {
-            config.MapHttpAttributeRoutes();
+            config.EnableHypermedia()
+                .RegisterDefaultTransform(new ReflectiveTransform());
```

Comment:
To be upfront, I don't fully understand what `ReflectiveTransform()` is doing here. I was modeling this set up based upon the Hypermedia sample.